fpx-lib/src/api/handlers/spans.rs

```diff
@@ -101,3 +101,40 @@ impl From<DbError> for ApiServerError<SpanListError> {
         ApiServerError::CommonError(CommonError::InternalServerError)
     }
 }
+
+#[tracing::instrument(skip_all)]
+pub async fn span_delete_handler(
+    State(store): State<BoxedStore>,
+    Path((trace_id, span_id)): Path<(String, String)>,
+) -> Result<StatusCode, ApiServerError<SpanDeleteError>> {
+    let tx = store.start_readonly_transaction().await?;
+
+    hex::decode(&trace_id)
+        .map_err(|_| ApiServerError::ServiceError(SpanDeleteError::InvalidTraceId))?;
+    hex::decode(&span_id)
+        .map_err(|_| ApiServerError::ServiceError(SpanDeleteError::InvalidSpanId))?;
```

Comment:
would be cool to write a custom extractor so this wouldnt have to be repeated for every route we use those on

Reply:
I'll add a type for it, that can do the conversion :+1:

Reply:
https://linear.app/fiberplane/issue/FP-4039/create-typed-traceidspanid